Tools/boringNetworkTool/main.py

```diff
@@ -0,0 +1,83 @@
+import socket
+import sys
+import time
+import json
+
+
+class Message:
+    def __init__(self, data):
+        self.data = data
+
+    def to_string(self) -> str:
+        return "%s \n" % str(self.data)
+
+
+def create_messages(raw_messages):
+    msgs = []
+    for message in raw_messages:
+        msgs.append(Message(data=message["data"]))
+    return msgs
+
+
+def send_with_udp(msgconfig, host, port):
+    try:
+        s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    except socket.error:
+        print('Failed to create socket')
+        sys.exit()
+    msgs = create_messages(msgconfig["config"]["messages"])
+    msg_frequency = msgconfig["config"]["messageFrequency"]
+    for message in msgs:
+        time.sleep(1.0 / msg_frequency)
+        try:
+            # Set the whole string
+            s.sendto(bytes(message.to_string(), 'utf-8'), (host, port))
+            print("message sent: %s" % message.to_string())
+            # receive data from client (data, addr)
+            if msgconfig["config"]["sendContinuous"] is False:
```

Comment:
We are probably gonna want to properly send the messages repetitively indefinitely